src/components/chatbox/chatbox.module.css

```diff
@@ -105,6 +105,24 @@
 	background: transparent;
 }
 
+/* This should be something like visually-hidden or sr-only class if it already exists */
+
+/* Basically we want to give the user with a screen reader some extra instructions */
+
+/* Ideally this would be nice for all users, so try to have useful help text visible to everyone if you can */
+.userInstructions {
+	position: absolute !important;
+	width: 1px !important;
+	height: 1px !important;
+	margin: -1px !important;
+	padding: 0 !important;
+	overflow: hidden !important;
+	white-space: nowrap !important;
+	border: 0 !important;
+	clip: rect(1px, 1px, 1px, 1px) !important;
+	clip-path: inset(50%) !important;
+}
```

Comment:
This is a common "visually hidden" or "screen-reader only" bit of CSS; do you have this as a helper class already available anywhere else?

Reply:
Yes we do! [The `g-screen-reader-only` class](https://github.com/hashicorp/dev-portal/blob/main/src/pages/style.css#L148-L166)